crates/sui-framework/packages/sui-framework/sources/balance.move

```diff
@@ -140,3 +142,19 @@ public fun destroy_for_testing<T>(self: Balance<T>): u64 {
 public fun create_supply_for_testing<T>(): Supply<T> {
     Supply { value: 0 }
 }
+
+#[allow(unused_function)]
+fun send_to_account<T>(balance: Balance<T>, recipient: address) {
+    let Balance { value } = balance;
+    let accumulator = accumulator::get_accumulator_field_address<Balance<T>>(recipient);
+    accumulator::emit_deposit_event<Balance<T>>(accumulator, recipient, value);
+}
+
+#[allow(unused_function)]
+fun withdraw_from_account<T>(amount: u64, ctx: &TxContext): Balance<T> {
```

Comment:
I'll end up renaming these, as a heads up--if the string needs to be stored anywhere